src/components/box/Box.tsx

```diff
@@ -47,9 +47,9 @@ const StyledBox = styled(GnuiContainer)<BoxProps & SpaceProps>`
   color: ${theme.color.text.text01};
   background: ${theme.color.background.ui01};
   font-weight: ${theme.fontWeights.regular};
-  box-shadow: ${theme.shadow.shadow02};
   transition: ${theme.transition};
   line-height: ${theme.lineHeight};
+  margin-bottom: ${theme.spacing.spacing02};
```

Comment:
This component is the main building block of Overwatch, do we really want to add margin here?
I feel like adding margins should be the responsibility of top-level components that use `Box`

Reply:
It should be the default margin-bottom but if you will not need this spacing you can just add spacing00